peekingduck/pipeline/nodes/input/visual.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2022 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Reads inputs from multiple visual sources
+- image or video file on local storage
+- folder of images or videos
+- online cloud source
+- CCTV or webcam live feed
```

Comment:
Did you intend for this to look like this in RTD? 

![image](https://user-images.githubusercontent.com/33995028/158769372-707a34ae-0b02-4f36-bf27-86c8e942862b.png)

Reply:
Fixed the formatting.